nemo_skills/evaluation/evaluator/livecodebench.py

```
@@ -92,25 +92,32 @@ async def execute_in_sandbox_with_retries(
-        LOG.warning(f"Sandbox is unavailable due to a network error: {type(e).__name__} - {e}")
-        return False
-    except Exception as e:
-        LOG.warning(f"An unexpected error occurred while checking sandbox availability: {e}")
+            result, _ = await execute_in_sandbox_with_retries(sandbox, 1, "true", language="shell", timeout=5)
+            if result.get("process_status") != "completed":
+                LOG.warning(f"Sandbox responded but command failed: {result.get('stderr')}")
+                return False
+
+            LOG.info("Sandbox connection successful. Sandbox is available.")
+            return True
+    except (httpx.NetworkError, Exception) as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)